webui/react/.eslintrc.js

```diff
@@ -59,6 +59,7 @@ module.exports = {
       beforeSelfClosing: 'always',
       closingSlash: 'never',
     } ],
+    'react/prop-types': 'off',
```

Comment:
**question:** isn't this a good thing to keep? maybe something was broken after eslint / plugins updated?